documentation/library_guide/cmake_support.rst

```diff
@@ -0,0 +1,108 @@
+CMake Support
+#############
+
+General Usage
+=============
+`CMake <https://cmake.org/cmake/help/latest/index.html>`_ is a cross-platform build system generator.  It can be used to generate build scripts which can then be used to build and link a users application.
```

Comment:
General note: We don't use a double space between sentences anymore. Please remove any double spaces between sentences. 

Please change "users" to "your".

Please update "integratration" to "integration".